### DISCUSSION TITLE
用AI加速保險理賠審核流程

### DISCUSSION THREAD

**Discussion by chia-kai-chang** [1101-GS3073-C/Python-Community-GS3073-C#111] [General]
參考影片：https://youtu.be/-m68OIdPqKI

**Comment by bonny123865** [comment]
那個人力跟AI的處理資料速度相比，真的差了好多阿！
而且準確率還比人類高那麼多，之後會不會變成大量裁人的產業？！

**Comment by ghost** [comment]
已經可以預想未來人力資源會被大量的AI給取代掉 不禁開始反思我們有那些優點可以不被AI取代

**Comment by Shiba010** [comment]
看起來以後保險業務員也要被取代了...
